metrics/web/test/common/presentation/tooltip_popup/theme_data/tooltip_popup_theme_data_test.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:metrics/common/presentation/tooltip_popup/theme_data/tooltip_popup_theme_data.dart';
+
+// https://github.com/platform-platform/monorepo/issues/140
+// ignore_for_file: prefer_const_constructors, avoid_redundant_argument_values
+
+void main() {
+  group("TooltipPopupThemeData", () {
+    test(
+      "creates an instance with the default background color if the color is not specified",
+      () {
+        final themeData = TooltipPopupThemeData();
+
+        expect(themeData.backgroundColor, isNotNull);
+      },
+    );
+
+    test(
+      "creates an instance with the default background color if the given color is null",
+      () {
+        final themeData = TooltipPopupThemeData(backgroundColor: null);
+
+        expect(themeData.backgroundColor, isNotNull);
+      },
+    );
+
+    test(
+      "creates an instance with null text style if the parameters are not specified",
```

Comment:
It seems like this test is redundant because it tests the default behavior of the class constructor.